### PR TITLE
ApplyOnnxModel sample

### PR DESCRIPTION
**Pull request by ganik**
part of #1209

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs [hunk 1]
```diff
@@ -6,10 +6,10 @@
 
 namespace Samples.Dynamic
 {
-    public static class OnnxTransformExample
+    public static class ApplyOnnxModel
     {
         /// <summary>
-        /// Example use of OnnxEstimator in an ML.NET pipeline
+        /// Example use of ApplyOnnxModel in an ML.NET pipeline
         /// </summary>
```
Comment (shmoradims) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs]: let's delete this <summary> (it's stating the obvious and we don't have it for other samples)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs [hunk 2]
```diff
@@ -18,29 +18,17 @@ public static void Example()
             // Microsoft.ML.Onnx.TestModels nuget.
             var modelPath = @"squeezenet\00000001\model.onnx";
```
Comment (shmoradims) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs]: Please verify that this statement is true:

1) Is the model.onnx in the github link the same as

**Comment by shmoradims** [comment]
>         /// </summary>

we're using regular // comments instead of /// (because the examples get pasted verbatim, and no xml is parsed)

---
Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs:59 in 1011013. [](commit_id = 10110131fae1590c6043ca3613c84fb3c2a29347, deletion_comment = False)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs (hunk 2 above)
Comment (ganik) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs]: yep its there

---
In reply to: [275814495](https://github.com/dotnet/machinelearning/pull/3349#discussion_r275814495) [](ancestors = 275814495)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs [hunk 3]
```diff
@@ -66,25 +50,25 @@ public static void Example()
             // ----------
         }
 
-        /// <summary>
-        /// inputSize is the overall dimensions of the model input tensor.
-        /// </summary>
+        // <summary>
```
Comment (shmoradims) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs]: i mean let's delete the xml *tags* like <summary> and just keep the comment text:

// inputSize is the overall dimensions of the model input tensor.

**Pull request merged**